spec/spec_helper.rb

```diff
@@ -21,7 +21,9 @@
 backtrace_filter.add_filter(%r{gems/railties})
 backtrace_filter.add_filter(%r{tapioca/helpers/test/})
 
-Minitest::Reporters.use!(SpecReporter.new(color: true), ENV, backtrace_filter)
+unless ENV["RM_INFO"]
```

Comment:
I'd rather not special case RubyMine here. I'd be open to changing the default reporter, though, since our custom reporter doesn't work that well.

Reply:
Let's try not using it at all, and see what happens?

Reply:
Left is without, right is with.

I see 2 slight benefits to using this SpecReporter:

1. The passing tests are summarized more tersely with simple green dots (though with less detail, like the name and runtime)
2. There's a backtrace filter applied

The stock one is more detailed, and perhaps better overall, but that's subjective

<img width="1913" alt="image" src="https://github.com/user-attachments/assets/f4669a30-4b73-4303-af99-e2f5f0126d59">